extensions/amp-story/1.0/test/test-localization.js

```diff
@@ -67,8 +67,8 @@ describes.fakeWin('localization', {}, env => {
     });
 
     it('should have language fallbacks', () => {
```

Comment:
We should add another test for the new case that tests a language code not being specified and defaulting to English

Reply:
That makes sense! I added a test, so let me know if you meant something else!